proof/refine/AARCH64/ArchArchAcc_R.thy

```diff
@@ -5,12 +5,10 @@
  * SPDX-License-Identifier: GPL-2.0-only
  *)
 
-(*
-  Lemmas on arch get/set object etc
-*)
+(* Arch-specific lemmas on arch get/set object etc *)
 
 theory ArchAcc_R
-imports SubMonad_R
+imports ArchArchAcc_R
```

Comment:
If we want to clean up the history, this should probably be changed to
```
theory ArchArchAcc_R
imports ArchAcc_R
```
here in the rename commit instead of in the following arch-split commit.

Reply:
good point, will try to untangle